### DISCUSSION TITLE
Connection refused with Python client

### DISCUSSION THREAD

**Discussion by fccoelho** [geldata/gel#3783] [Q&A] [answered]
Hi I am trying edgedb 1.3 with python client

I set up a database following the quick start and ended up with a `edgedb` database
after creating a simple schema and verifying that it worked in the CLI, I tried to insert data into the database using the Python client.

```python
client = edgedb.create_client('edgedb://edgedb@localhost/edgedb')
```
and then when I try to run the inserts with `client.query("...")`, I get the following error.

```python
ClientConnectionFailedTemporarilyError: [Errno 111] Connection refused
	After 271 attempts in 30.1 sec
	Is the server running on host 'localhost' and accepting 
	TCP/IP connections on port 5656?
```

**Comment by elprans** [comment, marked as answer]
Remove the DSN: `client = edgedb.create_client()`.  Local development instances run on arbitrary ports, so you rely on address auto-discovery.

**Reply by fccoelho** [reply]
Thanks! it worked.
